Lib/site.py

```diff
@@ -74,6 +74,7 @@
 import builtins
 import _sitebuiltins
 import io
+import sysconfig
```

Comment:
sysconfig creates tons of Python objects and is slow to import. I would prefer to avoid it at startup.

Would it be possible to move the logic into a smaller module which does exactly what you need, and reuse this module in sysconfig? See _sitebuiltins for example.

Reply:
Yes, I think this should be perfectly doable. It also stacks up another reason to implement @zooba's suggestion of making sysconfig a directory and installing the vendor config as a submodule, to prevent shadowing it.

Reply:
> It also stacks up another reason to implement @zooba's suggestion of making sysconfig a directory and installing the vendor config as a submodule, to prevent shadowing it.

Maybe I’m misunderstanding, but are you hinting to put the smaller module in a subpackage? Since that won’t work because `import sysconfig.bare` implicitly imports `sysconfig` as well so unless you move all names out of `sysconfig` (not an option since it breaks all existing code) this will not help import time.

Reply:
Right :facepalm: